drv/lpc55-gpio-api/src/lib.rs

```diff
@@ -15,7 +16,7 @@ use zerocopy::{AsBytes, FromBytes};
 // goes 0 - 64
 cfg_if::cfg_if! {
     if #[cfg(any(target_board = "lpcxpresso55s69"))] {
-        #[derive(Copy, Clone, Debug, FromPrimitive)]
+        #[derive(Copy, Clone, Debug, FromPrimitive, AsBytes, Serialize, Deserialize)]
```

Comment:
I don't think you actually need `Serialize` / `Deserialize` here, now that we're not using `ssmarshal`.  With the `derives` removed, you can then remove the `serde` import altogether and eliminate it from the `Cargo.toml`.